examples/1d_stencil/1d_stencil_1_omp.cpp

```diff
@@ -13,6 +13,7 @@
 // parallelizing the inner loop.
 
 #include <hpx/config/defines.hpp>   // avoid issues with Intel14/libstdc++4.4 nullptr
+#include <hpx/config.hpp> // Include before any boost header
```

Comment:
The first #include could be removed now. Including `<hpx/config.hpp>` should be sufficient.